client/components/swipeable/index.js

```diff
@@ -167,8 +180,13 @@ export const Swipeable = ( {
 
 			const dragPosition = getDragPositionAndTime( event );
 			const delta = dragPosition.x - dragStartData.x;
+			const absoluteDelta = Math.abs( delta );
 			const offset = getOffset( currentPage ) + delta;
 
+			if ( absoluteDelta < 3 ) {
```

Comment:
Could you add comment for the meaning of 3? Or we might not need this as it won't met the threshold?